examples/memory.rs

```diff
@@ -71,8 +71,10 @@ fn main() -> anyhow::Result<()> {
     let mem_size: TypedFunction<(), i32> = instance
         .exports
         .get_typed_function(&mut store, "mem_size")?;
+    #[cfg(not(feature = "wamr"))]
```

Comment:
Why not in wamr?

Reply:
Because the code down below makes use of the functions only conditionally:
https://github.com/wasmerio/wasmer/blob/b4bff3c554876b4300a28b102fb508ff2eb62f02/examples/memory.rs#L112-L145